package.json

```diff
@@ -54,7 +54,7 @@
     },
     "main": "src/main.js",
     "scripts": {
-        "testserver": "`npm bin`/http-server -p ${VELLUM_PORT:-8088}",
+        "testserver": "node_modules/.bin/http-server -p ${VELLUM_PORT:-8088}",
```

Comment:
It's at least middle-aged. I'm on 10 and get an error:
```
[Vellum] (master) $ npm --version
10.8.1
[Vellum] (master) $ npm bin
Unknown command: "bin"

To see a list of supported npm commands, run:
  npm help
```

Reply:
In places where commands need to be run that are not part of package.json, [`npm exec`](https://docs.npmjs.com/cli/v10/commands/npm-exec) or `npx` seem to be the recommended replacement. However, in this context I'm not sure we need any of that. This works in my local environment
```suggestion
        "testserver": "http-server -p ${VELLUM_PORT:-8088}",
```

Reply:
Is it possible you installed `http-server` globally at some point? When I was setting up my Vellum environment last week (new laptop since the last time I worked on vellum), I couldn't run just `http-server` until I installed the package globally. I just deleted the `node_modules` directory and tried again to see if it was really necessary, and I think it is:

```
[Vellum] (jls/drop-MediaUploader) $  rm -rf node_modules
[Vellum] (jls/drop-MediaUploader) $ ls node_modules/http-server
ls: node_modules/http-server: No such file or directory
[Vellum] (jls/drop-MediaUploader) $ yarn install --frozen-lockfile
yarn install v1.22.21
(node:36307) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 🔨  Building fresh packages...
warning Your current version of Yarn is out of date. The latest version is "1.22.22", while you're on "1.22.21".
info To upgrade, run the following command:
$ curl --compressed -o- -L https://yarnpkg.com/install.sh | bash
✨  Done in 39.73s.
[Vellum] (jls/drop-MediaUploader) $ ls node_modules/http-server
LICENSE         README.md       bin             doc             lib             node_modules    package.json
[Vellum] (jls/drop-MediaUploader) $ http-server
zsh: command not found: http-server
[Vellum] (jls/drop-MediaUploader) $ npm install -g http-server

added 46 packages in 2s

15 packages are looking for funding
  run `npm fund` for details
[Vellum] (jls/drop-MediaUploader) $ http-server
Starting up http-server, serving ./

http-server version: 14.1.1
...
```

Reply:
I forgot about your first sentence. `npx exec http-server` and `npx http-server` do both work for me locally even when the package is only installed locally, thank you for pointing those out.

Reply:
I verified that `http-server` was not on my path, and `npx` does not seem to be necessary in package.json. Does that not work for you? My guess is that `npx http-server` is only necessary when running it directly on the command line, but it's not needed in the context of a command in package.json.

```sh
$ which http-server
/usr/bin/which: no http-server in (...PATH...)

$ http-server
bash: http-server: command not found...

$ grep testserver package.json 
        "testserver": "http-server -p ${VELLUM_PORT:-8088}",

$ npm run testserver

> vellum@1.6.0 testserver
> http-server -p ${VELLUM_PORT:-8088}

Starting up http-server, serving ./
...
^Chttp-server stopped.
```